BGRSCRPT/iocBoot/iocBGRSCRPT-IOC-01/runIOC.bat

```diff
@@ -2,12 +2,28 @@
 setlocal
 set MYDIRBLOCK=%~dp0
 call C:\Instrument\Apps\EPICS\config_env_base.bat
-
```

Comment:
instead of hard coding  `C:\Instrument\Apps\EPICS` in the file  i think
`call %~dp0..\..\..\..\..\config_env_base.bat` will work